tests/test-app/test_app.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: load-balancer-example
+  name: hello-world
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: load-balancer-example
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: load-balancer-example
+    spec:
+      runtimeClassName: kontain
+      nodeSelector:
+        sandbox: kontain
+      containers:
+        - image: docker.io/ezleka/node_test
```

Comment:
Is this the permanent source of the test image?

Reply:
this is just my test app. It can be dweb or anything else entirely. This is just a very simple node app for testing

Reply:
I have created new public container registry with authorized anonymous pull for as to keep test images. We can also use that registry to push snapshot images while testing